realm_settings_general_tab_test.spec fails randomly

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

After running user_fed_ldap_hardcoded_mapper_test.spec.ts, running realm_settings_general_tab_test.spec.ts fails.

See https://github.com/keycloak/keycloak/actions/runs/9055714397/job/24877310977?pr=29465

### Version

main

### Regression

- [ ] The issue is a regression

### Expected behavior

test passing always

### Actual behavior

test fails

### How to Reproduce?

Run the cypress tests in the order above

### Anything else?

_No response_

https://github.com/keycloak/keycloak/actions/runs/9285154296/job/25549341766?pr=28518#step:11:205

```
Realm settings general tab tests
    (Attempt 1 of 4) Test all general tab switches
    (Attempt 2 of 4) Test all general tab switches
    (Attempt 3 of 4) Test all general tab switches
    1) Test all general tab switches
    ✓ Fail to set Realm ID to empty (1778ms)
    ✓ Modify Display name (2189ms)
    ✓ Check Display name value (1504ms)
    ✓ Modify front end URL (1964ms)
    ✓ Select SSL all requests (1863ms)
    ✓ Verify SSL all requests displays (1555ms)
    ✓ Select SSL external requests (1756ms)
    ✓ Verify SSL external requests displays (1494ms)
    ✓ Select SSL None (1759ms)
    ✓ Verify SSL None displays (1543ms)
    ✓ Check Access Endpoints OpenID Endpoint Configuration link (1490ms)
    ✓ Access Endpoints OpenID Endpoint Configuration link (1605ms)
    ✓ Check if Access Endpoints SAML 2.0 Identity Provider Metadata link exists (1534ms)
    ✓ Access Endpoints SAML 2.0 Identity Provider Metadata link (1511ms)
    ✓ Verify 'Revert' button works (1923ms)


  15 passing (3m)
  1 failing

  1) Realm settings general tab tests
       Test all general tab switches:
     AssertionError: Timed out retrying after 30000ms: Expected to find element: `.pf-v5-c-alert__title`, but never found it. Queried from:

              > cy.findByTestId(global-alerts)
      at Masthead.checkNotificationMessage (webpack://admin-ui/./cypress/support/pages/admin-ui/Masthead.ts:90:0)
      at Context.eval (webpack://admin-ui/./cypress/e2e/realm_settings_general_tab_test.spec.ts:44:0)
```

https://github.com/keycloak/keycloak/assets/38039883/98f177a7-ab9d-4da9-95d6-28749550495f

--- a/js/apps/admin-ui/cypress/e2e/realm_settings_general_tab_test.spec.ts
+++ b/js/apps/admin-ui/cypress/e2e/realm_settings_general_tab_test.spec.ts
@@ -49,12 +49,9 @@ describe("Realm settings general tab tests", () => {
     sidebarPage.waitForPageLoad();
 
     // Disable realm
-    const loadName = `load-disabled-${uuid()}`;
-    cy.intercept({ path: "/admin/realms/*", times: 1 }).as(loadName);
     realmSettingsPage.toggleSwitch(`${realmName}-switch`, false);
     realmSettingsPage.disableRealm();
     masthead.checkNotificationMessage("Realm successfully updated", true);
-    cy.wait(`@${loadName}`);
     sidebarPage.waitForPageLoad();
 
     // Re-enable realm

